chia/types/eligible_coin_spends.py

```diff
@@ -326,20 +335,15 @@ async def process_fast_forward_spends(
             new_coin_spends.append(new_coin_spend)
         if len(ff_bundle_coin_spends) == 0:
             # This item doesn't have any fast forward coins, nothing to do here
-            return
+            return bundle_coin_spends
         # Update the mempool item after validating the new spend bundle
         new_sb = SpendBundle(
             coin_spends=new_coin_spends, aggregated_signature=mempool_item.spend_bundle.aggregated_signature
         )
         # We need to run the new spend bundle to make sure it remains valid
```

Comment:
"but the results are expected to be the same", or something like that. Are they expected to be the same? are they the same?
Would it make sense to assert that?
I suppose there a risk some coin ID may have changed in the rebase, but I would expect most conditions to be the same as `mempool_item.conds. would you?

Reply:
Results are not expected to be exactly the same. They're mostly the same but the changes happen in additions and removals, and they are related to the rebased coins where parents, solutions and IDs...etc change.